openstack/clustering/v1/clusters/requests.go

```diff
@@ -95,3 +95,48 @@ func List(client *gophercloud.ServiceClient, opts ListOptsBuilder) pagination.Pa
 		return ClusterPage{pagination.LinkedPageBase{PageResult: r}}
 	})
 }
+
+// UpdateOpts implements UpdateOpts
+type UpdateOpts struct {
+	Config      string                 `json:"config,omitempty"`
+	Name        string                 `json:"name,omitempty"`
+	ProfileID   string                 `json:"profile_id,omitempty"`
+	Timeout     int                    `json:"timeout,omitempty"`
```

Comment:
Is a timeout value of `0` valid? If so then this should be `*int`.

Reply:
Should allow a Timeout of 0. Made changes